packages/shared/components/MenuSshLogin/MenuSshLogin.tsx

```diff
@@ -100,7 +101,7 @@ export const LoginItemList = ({ logins, onClick, onKeyPress }) => {
         key={key}
         px="2"
         mx="2"
-        as={NavLink}
+        as={url ? NavLink : null}
```

Comment:
In teleterm we don't have classical routing, so rendering `NavLink` causes errors (that it should be used in context of `Router`)